modules/cnv-cloning-pvc-of-vm-disk-into-new-datavolume.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume.adoc
+
+[id="cnv-cloning-pvc-of-vm-disk-into-new-datavolume_{context}"]
+= Cloning the PersistentVolumeClaim of a virtual machine disk into a new DataVolume
+
+You can clone a PVC of an existing virtual machine disk into a new DataVolume.
+The new DataVolume can then be used for a new virtual machine.
+
+[NOTE]
+====
+When a DataVolume is created independently of a virtual machine, the lifecycle
+of the DataVolume is independent of the virtual machine. If the virtual machine
+is deleted, neither the DataVolume nor its associated PVC will be deleted.
+====
+
+.Prerequisites
+* A PVC of an existing virtual machine disk. The associated virtual machine must
+be powered down, or the clone process will be queued until the PVC is available.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. Examine the virtual machine disk you want to clone to identify the name and
+namespace of the associated PVC.
+
+. Create a YAML file for a DataVolume object that specifies the name of the
+new DataVolume, the name and namespace of the source PVC, and the size of the new
+DataVolume.
++
+For example:
++
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: cloner-datavolume <1>
+spec:
+  source:
+    pvc:
+      namespace: "<source-namespace>" <2>
+      name: "<my-favorite-vm-disk>" <3>
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 500Mi <4>
+----
+<1> The name of the new DataVolume.
+<2> The namespace where the source PVC exists.
+<3> The name of the source PVC.
+<4> The size of the new DataVolume. You must allocate enough space, or the
+cloning operation fails. The size must be the same or larger as the source PVC.
+
+. Start cloning the PVC by creating the DataVolume:
++
+----
+$ oc create -f <cloner-datavolume>.yaml
+----
++
+[NOTE]
+====
+DataVolumes prevent a virtual machine from starting before the PVC is prepared,
+so you can create a virtual machine that references the new DataVolume while the
+PVC clones.
+====
```

Comment:
Is there a good next step for this procedure?

Reply:
I'm not sure. It's optional to create a VM at that point, but that's the next step for some customers.